http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -62,6 +64,26 @@ RELAY_CONNECTIONSTRING="Endpoint=sb://${RELAY_NAME}.servicebus.windows.net/;Shar
 
 /anaconda/envs/py38_default/bin/pip3 install seaborn
 
+# Update rbase
+
+echo "Y"|sudo apt install --no-install-recommends r-base
+
+#Update kernel list
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove sparkkernel
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove sparkrkernel
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove pysparkkernel  
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove spark-3-python 
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove julia-1.6
+
+echo "Y"| /anaconda/envs/py38_default/bin/pip3 install ipykernel
+
+echo "Y"| /anaconda/envs/py38_default/bin/python3 -m ipykernel install 
```

Comment:
Ah - is this the key thing that fixes the earlier issue?

Reply:
> Ah - is this the key thing that fixes the earlier issue with the phantom `Python 3` kernel?

Yes :)